Goal: Change cyrus user and group in templates

Correct permissions on /etc/imapd.group and other cyrus files
(http://bugs.debian.org/581757)

Author: Mathieu Parent <math.parent@gmail.com>

Upstream status: Already committed on TRUNK

Index: kolabd-2.2.3-20091217/dist_conf/debian
===================================================================
--- kolabd-2.2.3-20091217.orig/dist_conf/debian	2010-06-17 20:51:45.000000000 +0200
+++ kolabd-2.2.3-20091217/dist_conf/debian	2010-06-17 20:52:02.000000000 +0200
@@ -69,8 +69,8 @@
 
 imap_confdir=${sysconfdir}
 imap_confperm=0640
-imap_usr=root
-imap_grp=root
+imap_usr=cyrus
+imap_grp=mail
 imap_masterlogfile=${localstatedir}/log/mail.log
 imap_misclogfile=${localstatedir}/log/mail.log
 imap_statedir=${localstatedir}/lib/cyrus
